apps/src/lib/util/audioApiDropletConfig.js

```diff
@@ -32,6 +33,19 @@ const dropletConfig = {
       0: () => getAssetDropdown('audio')
     },
     assetTooltip: {0: chooseAsset.bind(null, 'audio')}
+  },
+  playSpeech: {
```

Comment:
Do we need to hide this block behind an experiment?

Reply:
it's behind an experiment in applab and gamelab and it needs to be added to a lab's specific dropletConfig to show up.